docs/destinations/warehouses-lakes/index.md

```diff
@@ -66,24 +61,30 @@ All lake loaders are micro-batching.
 | ---- | -------- | ---------------- | ------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| S3   | Delta    | Athena           | [Lake Loader](/docs/api-reference/loaders-storage-targets/lake-loader/index.md) | Early release                                                                                                                                                                                                                                  |
-| S3   | Iceberg  | Athena, Redshift | [Lake Loader](/docs/api-reference/loaders-storage-targets/lake-loader/index.md) | Early release                                                                                                                                                                                                                                  |
-| S3   | TSV/JSON | Athena           | [S3 Loader](/docs/api-reference/loaders-storage-targets/s3-loader/index.md)     | Only recommended for use with [RDB Batch Transformer](/docs/api-reference/loaders-storage-targets/snowplow-rdb-loader/transforming-enriched-data/spark-transformer/index.md) or for [failed events](/docs/fundamentals/failed-events/index.md) |
+| S3   | Delta    | Athena, <s>Databricks</s>           | [Lake Loader](/docs/api-reference/loaders-storage-targets/lake-loader/index.md) | Production-ready                                                                                                                                                                                                                                  |
+| S3   | Iceberg  | Athena, Redshift | [Lake Loader](/docs/api-reference/loaders-storage-targets/lake-loader/index.md) | Production-ready                                                                                                                                                                                                                                  |
+| S3   | TSV/JSON | Athena           | [S3 Loader](/docs/api-reference/loaders-storage-targets/s3-loader/index.md)     | Only recommended for use with [RDB Batch Transformer](/docs/api-reference/loaders-storage-targets/snowplow-rdb-loader/transforming-enriched-data/spark-transformer/index.md) or for [raw failed events](/docs/fundamentals/failed-events/index.md) |
+
+:::tip
+
+Please note that currently the S3 _Delta_ loader is not compatible with Databricks. The loader uses [DynamoDB tables for mutually exclusive writes to S3](https://docs.delta.io/latest/delta-storage.html#multi-cluster-setup), a feature of Delta. Databricks, however, does not support this (as of July 2025). This means that it’s not possible to alter the data via Databricks (e.g. to run `OPTIMIZE` or to delete PII).
```

Comment:
> not possible to alter the data

I paused on this line, because the word "alter" makes me think of DDL statements, like `ALTER TABLE ADD COLUMN`.  But I don't have a suggestion for a better word.  And you said "alter the data" so that's ok.